src/Loader.php

```diff
@@ -367,7 +367,17 @@ public static function register_scripts() {
 		wp_register_script(
 			'wc-customer-effort-score',
 			self::get_url( 'customer-effort-score/index', 'js' ),
-			array(),
+			array(
+				'wp-components',
+				'wp-compose',
+				'wp-data',
+				'wp-element',
+				'wp-18n',
+				'wp-notices',
+				'classnames',
```

Comment:
I don't think we need to include the non-WordPress packages in this array (`classnames`, `prop-types`, `react-transition-group`).  I think those packages are bundled by Webpack instead?

Reply:
Thanks @adrianduffell , I took these out and fixed a typo (`wp-18n` vs `wp-i18n`) which was stopping this from working :-$